controllers/metal3data_controller_test.go

```diff
@@ -80,11 +80,11 @@ var _ = Describe("Metal3Data manager", func() {
 				}
 				if tc.m3d != nil && !tc.m3d.DeletionTimestamp.IsZero() {
 					if tc.releaseLeasesRequeue {
-						m.EXPECT().ReleaseLeases(context.TODO()).Return(baremetal.WithTransientError(errors.New(""), requeueAfter))
+						m.EXPECT().ReleaseLeases(context.Background()).Return(baremetal.WithTransientError(errors.New(""), requeueAfter))
```

Comment:
was changing the context from todo to background required by the new go version ?

Reply:
Yes, the function uses context background but the test was expecting context todo